tests/fake-tasks/pyproject.toml

```diff
@@ -8,9 +8,7 @@ authors = [
 readme = "README.md"
 license = {text ="BSD-3-Clause"}
 requires-python = ">=3.10,<3.13"
-dependencies = [
-    "pydantic >=2.0.0,<=2.8.2",
-]
+dependencies = ["pydantic >=2.0.0"]
```

Comment:
Didn't you want to pin pydantic to <= 2.11.2 (see https://github.com/fractal-analytics-platform/fractal-task-tools/issues/2#issuecomment-2782919067)?